### PR TITLE
 Adapt (fit) the width of the combo list to the content (for long id or name without wrapping)

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

File: src/components/ElementSearchDialog/element-search-dialog.js [hunk 1]
```diff
@@ -30,8 +31,17 @@ const useStyles = makeStyles({
         padding: '20px',
         paddingRight: '0px',
     },
+    popper: {
+        maxWidth: 'fit-content',
+    },
 });
 
+const useStyles = makeStyles(styles);
```
Comment (sBouzols) [inline src/components/ElementSearchDialog/element-search-dialog.js:39]: to avoid this.

File: src/utils/EquipmentType.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ import match from 'autosuggest-highlight/match';
 import parse from 'autosuggest-highlight/parse';
 import clsx from 'clsx';
 import React from 'react';
+import { LIGHT_THEME } from '../components/TopBar/TopBar';
```
Comment (sBouzols) [inline src/utils/EquipmentType.js:11]: Not sure we should create  direct dependency to TopBar component code here.
Maybe use :    
`import { LIGHT_THEME } from '../..';`    
which is a dependency to the commons-ui exports index (yes I know its exports from TopBar components but, you know what I mean...)
Actually we will adapt this later...

**Review by jonenst — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Approved but some comments
demo test (light and dark theme) OK
console outputs OK

File: src/utils/EquipmentType.js (hunk 2 above)
Comment (sBouzols) [reply src/utils/EquipmentType.js:11]: avoid import from '../..', it has terrible impacts over console warnings... don't know why.

**Pull request merged**